server/repos/movieRepo.js

```diff
@@ -32,43 +32,56 @@ async function postReview(data, movieId) {
     const currentMovie = await getMovieById(movieId);
     currentMovie.reviews.push(newReview);
     await currentMovie.save();
-    await updateTags(data, movieId);
+    await updateRatings(data, movieId);
     await updateFmScore(data, movieId);
+    await updateTags(movieId);
     return await currentMovie;
 }
 
-async function updateTags(data, movieId) {
+async function updateTags(movieId) {
```

Comment:
@iahelr @yarindeoh 
I wrote updateTags similar to updateRatings. but it is weird that we ask several times getMovieById instead of using the one we have. is there a reason for that?